spec/models/users/notification_setting_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe Users::NotificationSetting, type: :model do
+  let!(:user) { create(:user) }
+  let(:notification_setting) { described_class.find_by(user_id: user.id) }
```

Comment:
**Suggestion (non-blocking):** This also works and may be a bit more straightforward:

```suggestion
  let(:notification_setting) { user.notification_setting.reload }
```